virl2_client/event_handling.py

```diff
@@ -195,13 +265,15 @@ def _parse_element_created(self, event: Event) -> None:
                 event.data["interface_b"],
             )
         else:
-            _LOGGER.warning("Received an invalid element type (%s)", event.element_type)
+            _LOGGER.warning(f"Received an invalid element type ({event.element_type})")
```

Comment:
There are also annotations. And connector mappings (which really only have one generic event, but those aren't "invalid".
You could just ignore them with a maybe debug/info log.